modules/cactus-web/src/BrandBar/BrandBar.tsx

```diff
@@ -205,6 +290,190 @@ const StyledUserMenu = styled.div`
+        buttonRef.current?.focus()
+        break
+    }
+  }
+
+  const handleTriggerClick = () => {
+    toggle()
+  }
+
+  return (
+    <>
+      <DropdownButton
+        $isTiny={isTiny}
+        ref={buttonRef}
+        onClick={handleTriggerClick}
+        {...buttonProps}
+      >
+        <span>{label}</span>
+        <NavigationChevronDown aria-hidden ml={4} />
+      </DropdownButton>
+      <DropdownPopup
+        buttonRef={buttonRef}
+        popupRef={portalRef}
+        isOpen={expanded}
+        $isTiny={isTiny}
+        {...popupProps}
+        onKeyDown={handlePopupKeyDown}
+      >
+        {children}
+      </DropdownPopup>
+    </>
+  )
+}
+
+const BasePopup: React.FC<PopupProps> = ({ buttonRef, popupRef, isOpen, ...props }) => {
+  const isTiny = SIZES.tiny === useScreenSize()
+  const positionPortal = React.useCallback<
+    (popover: HTMLElement, target: HTMLElement | null) => void
+  >(
+    (popover, target) => {
+      if (!target || !popover || !buttonRef.current) {
+        return
+      }
+      const targetRect = target.getBoundingClientRect()
+      const popoverRect = popover.getBoundingClientRect()
+      const buttonWidth = buttonRef.current.clientWidth
+
+      const topPosition = getTopPosition(targetRect, popoverRect)
+      popover.style.top = topPosition.top
+      popover.style.left = isTiny ? '0px' : `${targetRect.left}px`
+      popover.style.minWidth = `${buttonWidth}px`
+      if (isTiny) {
+        popover.style.right = '0px'
+      } else {
+        popover.style.maxWidth = `${window.innerWidth - targetRect.left}px`
+      }
+    },
+    [buttonRef, isTiny]
+  )
+  usePositioning({
+    anchorRef: buttonRef,
+    ref: popupRef,
+    visible: isOpen,
+    position: positionPortal,
+    updateOnScroll: true,
+  })
+  return <FocusLock ref={popupRef} {...props} />
+}
+
+const StyledDropdownRow = styled(Flex)`
+  border-top: ${(p) => border(p.theme, 'lightContrast')};
+  border-bottom: ${(p) => border(p.theme, 'lightContrast')};
+`
+
+const DropdownButton = styled.button<{ $isTiny: boolean }>`
+  box-sizing: border-box;
+  cursor: pointer;
+  font-weight: 600;
+  margin: 0;
+  display: flex;
+  align-items: center;
+  flex-wrap: nowrap;
+  background-color: transparent;
+  ${(p) => textStyle(p.theme, 'body')};
+  padding: ${(p) => p.theme.space[4]}px;
+  border: 0;
+  border-left: ${(p) => border(p.theme, 'lightContrast')};
+  border-right: ${(p) => border(p.theme, 'lightContrast')};
+  outline: none;
+
+  &[aria-expanded='true'] {
+    color: ${(p) => p.theme.colors.callToAction};
+
+    ${NavigationChevronDown} {
+      transform: scaleY(-1);
+    }
+  }
+
+  &:focus {
+    ${(p) => !p.$isTiny && insetBorder(p.theme, 'callToAction')};
+    color: ${(p) => p.theme.colors.callToAction};
+  }
+
+  &:hover {
+    color: ${(p) => p.theme.colors.callToAction};
+  }
+
+  & svg {
+    font-size: 12px;
+  }
+`
+
+const DropdownPopup = styled(BasePopup)`
+  display: ${(p) => (p.isOpen ? 'block' : 'none')};
```

Comment:
I'd suggest using the `aria-hidden` attribute instead of passing an `isOpen` prop solely for this style:
```suggestion
  &[aria-hidden='true'] {
    display: none;
  }
```